suite-common/wallet-utils/src/accountUtils.ts

```diff
@@ -352,6 +352,12 @@ export const getAllAccounts = (deviceState: string | typeof undefined, accounts:
 export const getNetwork = (symbol: string): Network | null =>
     NETWORKS.find(c => c.symbol === symbol) || null;
 
+export const getAccountNetwork = ({
```

Comment:
Nice to have: incorporate this into `getNetwork` and refactor code which does the same without using the util.

Reply:
I've done something like this as a separate draft PR #7418, because I didn't want to pollute this PR.